support-frontend/app/controllers/Application.scala

```diff
@@ -496,7 +496,7 @@ class Application(
   def productCheckoutRouter(countryGroupId: String) = MaybeAuthenticatedAction { implicit request =>
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
     val geoData = request.geoData
-    val serversideTests = generateParticipations(Nil)
+    val serversideTests = generateParticipations(List("lazyLoading"))
```

Comment:
Presume this function creates the `control` and `variant` for use later.

Reply:
Correct yes, it's a 50/50 chance of assigning control or variant